### PR TITLE
Remove alpha value from MSButton colors and use the property primary …

### PR DESCRIPTION
**Pull request by harrieshin**
…theme color.
Before | After
------------ | -------------
before_light  | after_light
before_dark | after_dark

### CODE REVIEW

File: ios/OfficeUIFabric/Controls/MSButton.swift [hunk 1]
```diff
@@ -129,6 +129,15 @@ open class MSButton: UIButton {
         proposedTitleLabelWidth = bounds.width - (contentEdgeInsets.left + contentEdgeInsets.right)
     }
 
+    public override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        if #available(iOS 13, *) {
+            if previousTraitCollection?.userInterfaceStyle != traitCollection.userInterfaceStyle {
+                updateBorderColor()
```
Comment (markavitale) [inline ios/OfficeUIFabric/Controls/MSButton.swift:136]: We should be able to handle this by just setting the color in updateLayer(), which is the Apple recommendation I believe.

Comment (markavitale) [reply ios/OfficeUIFabric/Controls/MSButton.swift:136]: We can probably get rid of all the methods updating colors and put them directly in updateLayer. Anywhere we were explicitly changing colors, just call setNeedsUpdateLayer.

Comment (harrieshin) [reply ios/OfficeUIFabric/Controls/MSButton.swift:136]: huh... isn't updateLayer a macOS thing?

Comment (markavitale) [reply ios/OfficeUIFabric/Controls/MSButton.swift:136]: Doh. Ignore my dopey mixing up of NSView and UIView calls. According to https://developer.apple.com/documentation/xcode/supporting_dark_mode_in_your_interface you should be fine with the traitCollectionDidChange location.

**Pull request merged**